docs/src/pages/system/styled/styled-pt.md

```diff
@@ -34,7 +34,6 @@ The utility can be used as a replacement for emotion's or styled-components' sty
    - `options.slot` (_string_ [optional]): If `Root`, it automatically applies the theme's `styleOverrides` & `variants`.
    - `options.overridesResolver` (_(props: object, styles: Record<string, styles>) => styles_ [optional]): Function that returns styles based on the props and the `theme.components[name].styleOverrides` object.
    - `options.skipVariantsResolver` (_bool_): Disables the automatic resolver for the `theme.components[name].variants`.
-   - `options.skipSx` (_bool_ [optional]): Disables the `sx` prop on the component.
```

Comment:
We should still have this option on the docs in case if developers want to disable the `sx` prop.